client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -522,7 +524,7 @@ export default function CampaignItemDetails( props: Props ) {
 								</div>
 							</div>
 						</div>
-						{ ( status === 'finished' || status === 'canceled' ) && (
+						{ canDisplayPaymentSection && (
```

Comment:
We need to change this condition because it is showing a 0 in the page. We can use a ternary with null here.

![Screenshot 2023-06-23 at 12 27 27](https://github.com/Automattic/wp-calypso/assets/1258162/168a1a60-3db3-4940-babb-e92c25ac58f2)

Reply:
whoops nice catch. A ternary will fix it. Thanks